.vscode/launch.json

```diff
@@ -18,8 +18,8 @@
       ],
       "outFiles": [
         "${workspaceRoot}/packages/vscode-extension/**/*.js",
-        "${workspaceRoot}/packages/language-support/**/*.cjs",
-        "${workspaceRoot}/packages/schema-poller/**/*.cjs"
+        "${workspaceRoot}/packages/language-support/**/*.js",
+        "${workspaceRoot}/packages/schema-poller/**/*.js"
```

Comment:
is this to get the vscode debugging to work better?

Reply:
I needed to do a manual build to get it to work, that's so I added it as a pre-launch task in the "cleanup turborepo" PR, but this seems like the better fix